### PR TITLE
chore(test): initialize e2e testing

### PR DESCRIPTION
**Pull request by iamdey**
+ implement protractor e2e framework
+ add an aria-label for user-menu
+ test actual features
+ mock api and configure CI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@
 dist
 node_modules
 npm-debug.log
+reports/
+screenshots/
```
Comment (ncoden) [inline .gitignore]: ?

Comment (iamdey) [inline .gitignore]: qu'elle est la question?

Comment (ncoden) [inline .gitignore]: why "screenshots" ?

Comment (iamdey) [inline .gitignore]: because tests saves screenshot on failure. The path can be changed in nightwatch.json

Comment (ncoden) [inline .gitignore]: I prefer a path for test-time content, for exemple `.test/`, like we have a `.build/` for build content, instead of the root

Comment (iamdey) [inline .gitignore]: +1

**Comment by iamdey** [comment]
nightwatchjs seems pretty unstable with angular since there is latency in http request and rendering.
We won't add pause before any action. Protractor does it well out of box for angular 1 & 2.

File: src/js/directive/layout/user-menu.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ export function LayoutUserMenuDirective() {
     template: `
       <ul class="nav navbar-nav navbar-right">
         <li class="dropdown">
-          <a  href class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
+          <a href class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="{{'header.menu.account'|translate}}">
```
Comment (ncoden) [inline src/js/directive/layout/user-menu.js]: -> in an other PR

Comment (iamdey) [inline src/js/directive/layout/user-menu.js]: this is required to validates the tests

**Comment by ncoden** [comment]
LGTM ;)

**Pull request merged**